blog/_posts/2017-09-8-railsgyn-lastblogpost.md

```diff
@@ -0,0 +1,59 @@
+---
+title: RailsGyn come back to reality.
+layout: post
+created_at: Fri September 22 2017
+permalink: blog/2017-09-22-RailsGyn-come-back-to-reality
+current: blog
+author: Amanda & Juliana
+twitter: RailsGyn
+categories:
+- student-posts
+---
+
+
+## All good things come to an end… OH GOD WHY!??!
+We are just shocked! We still cannot believe how fast these three months passed!
+We do not want this summer to finish!
+
+<img src="/img/blog/2017/cry.gif" alt="crying">
+<div align="center" div class="image-credits">That is exactly how we are now!</div>
+
+RGSoC was by far the best place ever that we already worked on. The environment was pure learning and assistance, we could implement a lot of cool features, we could meet a lot of nice people, we could discuss our ideas and we were heard. Certainly, we will keep contributing with the RGSoC team's app, because this program deserves the best management app that could exist and we want to work on it!
+
+<img src="/img/blog/2017/icons.png" alt="rgsoc">
+<div align="center" div class="image-credits">Design by Juliana</div>
+
+## What were our main achievements in this summer?
+
+We started the summer of code working on beginner friendly issues, however, on the very first days, one of our friends from TeamsApp asked for help in a milestone to automate the conference process to offer conference tickets to teams. It was a big change, with a lot of issues and a lot of models builds and refactors. She even though that it wasn’t possible to build for this summer, but we did it! We build almost all the related things to this milestone conclusion and when we saw it on production was proud.
+It was really surrounding construct such a complex issue in a really new project for us. We had to learn a lot in just some days, but worth every second when we saw everything working.
+That is was a challenge do with you, it makes you find out how far you can go.
```

Comment:
This sentence 'That is was a challenge do with you, it makes you find out how far you can go' looks bit odd. Can you rephrase it?